src/Options/Definitions.js

```diff
@@ -265,7 +265,7 @@ module.exports.ParseServerOptions = {
     "default": 10
   },
   "port": {
-    "env": "PARSE_SERVER_PORT",
+    "env": "PORT",
```

Comment:
this was dumbly competing with k8s when your service is named PARSE_SERVER, k8s exposed it as an env variable, but this is not a port (int) but the tcp URL for the service 🤕🔫